test/test_utils.cc

```diff
@@ -194,6 +194,16 @@ struct global_setup_and_teardown : Catch::EventListenerBase {
 		g_test_log_capture->print_log_if_nonempty(); // if so, this is likely due to a fatal signal
 		g_test_log_capture->clear();
 	}
+
+	void benchmarkPreparing(Catch::StringRef name) override {
+		log_level_before_benchmark = spdlog::get_level();
+		spdlog::set_level(spdlog::level::off);
```

Comment:
Maybe we should add a note here why this is important!

Reply:
Done.